src/ast/operator.rs

```diff
@@ -21,6 +21,12 @@ pub enum UnaryOperator {
     Plus,
     Minus,
     Not,
+    PGBitwiseNot,
+    PGSqrt,
+    PGCbrt,
```

Comment:
I think adding doc comments for the non-trivial operators is useful, as they are easily available on hover in many IDEs. E.g.
```
Cube root, e.g. `||/ 9` (PostgreSQL-specific)
```